pages/news/[slug].js

```diff
@@ -0,0 +1,106 @@
+import Head from 'next/head';
+import styles from '../../styles/News.module.css';
+import { useRouter } from 'next/router';
+import { Header } from '../../components/Header'
+
+export const News = ({ articles, pageNumber }) => {
+  const router = useRouter();
+  return articles.length ? (
+    <>
+      <Head>
+        <meta property="og:image" content={articles[0]?.urlToImage} />
+        <meta property="og:description" content={articles[0]?.description} />
+        <meta property="og:title" content={articles[0]?.title + ' and more!'} />
+      </Head>
+      <div className="page-container">
+        <Header />
+
+        <div className={styles.main}>
+          {articles.map((article, index) => (
+            <div key={index} className={styles.post}>
+              <h1 onClick={() => (window.location.href = article.url)}>{article.title}</h1>
```

Comment:
ooo what does `window.location.href = article.url` do? Is there a difference between a regular `<a>` with an href vs using an `onClick` for this?